sirepo/util.py

```diff
@@ -235,6 +238,14 @@ def split_comma_delimited_string(s, f_type):
     return [f_type(x) for x in re.split(r'\s*,\s*', s)]
 
 
+def url_safe_hash(value):
+    return pkcompat.from_bytes(
+        base64.urlsafe_b64encode(
```

Comment:
I think this should be simpler: hashlib.md5(pkcompat.to_bytes(value)).hexdigest()

There's no need for this to be compacted to base64 since it is a md5 hash, which is fixed size (32).

Reply:
791ce0e2